pom.xml

```diff
@@ -113,6 +128,17 @@ THE SOFTWARE.
             <version>2.0.1</version>
             <type>jar</type>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>matrix-project</artifactId>
+            <version>1.6</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci</groupId>
+            <artifactId>htmlunit</artifactId>
+            <version>2.6-jenkins-6</version>
```

Comment:
I think it's depended via jenkins-test-harness.
Why do you need this dependency?